tests/Command/PullRequest/PullRequestMergeCommandTest.php

```diff
@@ -67,26 +67,14 @@ class PullRequestMergeCommandTest extends BaseTestCase
 OET;
 
     const COMMAND_DISPLAY = <<<OET
-! [NOTE] Adding remote "gushphp" with "https://github.com/gushphp/gush.git".
```

Comment:
so because it ensures then it does not output these, perhaps we miss the test when it does not exist but it gets created, but i guess that is a bit harder. So i will merge :+1: